test/IexecWrapper.ts

```diff
@@ -0,0 +1,164 @@
+// SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber, ContractReceipt } from 'ethers';
+import { ethers } from 'hardhat';
+import config from '../config/config.json';
+import {
+    AppRegistry,
+    AppRegistry__factory,
+    DatasetRegistry,
+    DatasetRegistry__factory,
+    IexecAccessors__factory,
+    IexecInterfaceNative__factory,
+    IexecMaintenanceDelegate__factory,
+    RLC__factory,
+    WorkerpoolRegistry,
+    WorkerpoolRegistry__factory,
+} from '../typechain';
+import { IexecAccounts } from '../utils/poco-tools';
+import { extractEventsFromReceipt } from '../utils/tools';
+const DEPLOYMENT = config.chains.default;
+
+export class IexecWrapper {
+    proxyAddress: string;
+    accounts: IexecAccounts;
+
+    constructor(proxyAddress: string, accounts: IexecAccounts) {
+        this.proxyAddress = proxyAddress;
+        this.accounts = accounts;
+    }
+
+    /**
+     * Deposit value in iExec account.
+     * @param value The value to deposit.
+     * @param account Deposit value for an account.
+     */
+    async depositInIexecAccount(account: SignerWithAddress, value: number) {
```

Comment:
Maybe:
```
    if (['Native', 'native'].includes(DEPLOYMENT_CONFIG.asset)) {
        await IexecInterfaceNative__factory. [...] .deposit()
        return;
    }
    // token
    [...]
```

Reply:
I'm not 100% to understand, do you expect that [Rename config and set explicit switch case](https://github.com/iExecBlockchainComputing/PoCo/pull/60/commits/f2ea8d69df1ac8c5839d8251f56c027f4433597d)?

Reply:
I meant to use only one `if/return` instead of both `if/else`.
Switch is also OK for me (although a bit more verbose).